stats/src/main/java/io/airlift/stats/cardinality/DenseHll.java

```diff
@@ -272,28 +270,24 @@ public void insert(int bucket, int value)
         }
     }
 
-    private int getOverflow(int bucket)
+    private int findOverflowEntry(int bucket)
     {
         for (int i = 0; i < overflows; i++) {
             if (overflowBuckets[i] == bucket) {
```

Comment:
should that be some hash map instead?

Reply:
Possibly, but overflows are statistically infrequent and there's usually only a few of them. Scanning a few items may be faster than computing a hashcode.

Reply:
Right, could you just use array with number of buckets length? Would that help?

Reply:
That would be a lot of wasted space. Out of, say, 4096 buckets, there might only < 10 that overflow at any given point.